jinja_utils.py

```diff
@@ -59,17 +59,6 @@ def get_jinja_env(dir_path):
         os.path.dirname(__file__), dir_path))
     env = jinja2.Environment(autoescape=True, loader=loader)
 
-    def include_js_file(filepath):
-        """Include a raw JS file in the template without evaluating it."""
-        assert filepath.endswith('.js')
-        raw_file_contents = loader.get_source(env, filepath)[0]
-        if filepath == _OPPIA_MODULE_DEFINITION_FILE:
-            return jinja2.Markup(raw_file_contents)
-        else:
-            # Wrap the file in an immediately-invoked function expression
-            # (IIFE) to prevent pollution of the global scope.
-            return jinja2.Markup('(function() {%s})();' % raw_file_contents)
```

Comment:
@seanlip I'm a bit concerned about this. Getting rid of the IIFEs is definitely not a step in the right direction; I also forgot about this when we were planning the project. Any thoughts on workarounds?